rules/alerting.go

```diff
@@ -120,6 +120,8 @@ type AlertingRule struct {
 	annotations labels.Labels
 	// Time in seconds taken to evaluate rule.
 	evaluationDuration time.Duration
+	// Unix timestamp of last evaluation of rule
```

Comment:
Style nit: add a period at end of comment (we do sentence-style comments).

Reply:
Missed this one. I'll update :)